src/utils/transaction-utils.js

```diff
@@ -106,6 +106,43 @@ export const mapEchoTxResultToEth = (echoTx, blockNumber, txIndex, asset) => {
 	return ethereumTransaction;
 };
 
+/**
+ *
+ * @param {Object} echoTx
+ * @param blockNumber
+ * @param bloom
+ * @param logs
+ * @param transactionHash
+ * @param transactionIndex
+ * @return {*}
+ */
+export const mapEchoTxReceiptResultToEth = (echoTx, blockNumber, bloom, logs, transactionHash, transactionIndex) => {
+	const { operations } = echoTx;
+	const [[operationId, targetOperation]] = operations;
+
+	const ethereumTransactionReceipt = {};
+	ethereumTransactionReceipt.blockHash = encodeBlockHash(blockNumber);
+	ethereumTransactionReceipt.blockNumber = blockNumber;
+	ethereumTransactionReceipt.contractAddress = null;
```

Comment:
@adziaryba where is handling of setting of `contractAddress` if contract has been created?